src/Keboola/SklikExtractor/Api.php

```diff
@@ -133,7 +133,7 @@ protected function call($method, array $args = [])
         } while (true);
     }
 
-    public function getStats($userId, $campaignIdsBlock, $startDate, $endDate, $context = false)
+    public function getStats($userId, $campaignIdsBlock, $startDate, $endDate, $impressionShare, $context = false)
```

Comment:
I'm sorry, I probably haven't expressed it properly. I meant that it would be nice to have the `$impressionShare` parameter optional with default value (`false`). 

And if I may to point out, we use TDD approach for development and since this is open source repository, I would ask you to add test which gets stats with `impressionShare` set to `true`. Thanks